crates/ruma-signatures/src/functions.rs

```diff
@@ -74,6 +74,14 @@ fn allowed_content_keys_for(event_type: &str, version: &RoomVersionId) -> &'stat
             // TODO: Should we return an error for unknown versions instead?
             _ => &[],
         },
+        #[cfg(feature = "unstable-msc2870")]
+        "m.room.server_acl" => {
+            if version.as_str() == "org.matrix.msc2870" {
+                &["allow", "deny", "allow_ip_literals"]
+            } else {
+                &[]
+            }
+        }
```

Comment:
I think a match guard would be the more succint solution :)

```suggestion
        "m.room.server_acl" if version.as_str() == "org.matrix.msc2870" => &[
            "allow", "deny", "allow_ip_literals",
        ],
```

Reply:
Oh, I was trying to do this, but I couldn't find the right syntax